src/models/CoinDefinition.ts

```diff
@@ -352,7 +352,7 @@ export const ALL_COINS: CoinDefinition[] = [
     name: 'PancakeSwap',
     symbol: 'CAKE',
     blockchains: ['BSC'],
-    referrals: ['i0kx-z4kc'],
+    referrals: ['ofq5-w659'],
```

Comment:
@evan-cohen Can you verify that this code isn't yours either.

Reply:
not mine